### PR TITLE
refactor(user-profile): use styled-components (#534)

### PR DESCRIPTION
**Pull request by dillansimmons**
A little unclear what the best method for passing styling to the `ImageZoom` component based on if user was logged in.

### CODE REVIEW

File: src/components/user-profile.component.js [hunk 1]
```diff
@@ -16,85 +16,68 @@ type Props = {
...
+const Profile = styled.View`
+  flex: 3;
+  align-items: center;
+  justify-content: flex-end;
+`;
+const Title = styled.Text`
+  color: ${colors.white};
+  font-family: ${styledFonts.fontPrimaryBold};
+  font-size: ${normalize(16)};
+  margin-bottom: 2px;
```
Comment (chinesedfan) [inline src/components/user-profile.component.js]: We prefer unitless styles, except for shorthands properties.

File: src/components/user-profile.component.js [hunk 2]
```diff
@@ -120,29 +103,32 @@ export class UserProfile extends Component {
...
+                  {
+                    width: 75,
+                    height: 75,
+                    marginBottom: 20,
+                    borderRadius: 37.5,
+                  },
+                  (initialUser.type === 'User' || user.type === 'User') && {
+                    borderColor: colors.white,
+                    borderWidth: 2,
+                  },
```
Comment (chinesedfan) [inline src/components/user-profile.component.js]: @dillansimmons I think you can design a prop for StyledImageZoom. For example,

```
const StyledImageZoom = styled(ImageZoom)`
    // blabla
    ${({isUser}) => { /* additional styles */}}
`;
// in render
<StyledImageZoom isUser={initialUser.type === 'User' || user.type === 'User'}></StyledImageZoom>
```

**Comment by machour** [comment]
Hey @dillansimmons, friendly ping, are you still up for this? We're trying to close the styled-components migration 😄

**Comment by dillansimmons** [comment]
Hey @machour, yeah I've stalled out on this. I get the error: `Cannot create styled-component for component: undefined` when trying to extend the ImageZoom component. 

```
const StyledImageZoom = styled(ImageZoom)`
    width: 75;
    height: 75;
    margin-bottom: 20;
    border-radius: 37.5;
    ${({isUser}) => {
    `border-color: ${colors.white};
    border-width: 2;
    `}}
`;
```

Happy to let someone else take a shot at this, not a styled components expert by any means.

**Comment by chinesedfan** [comment]
> I get the error: Cannot create styled-component for component: undefined when trying to extend the ImageZoom component.

@dillansimmons I finished your work in #724. The error is due to the import sequence. And we are always glad to see your other new PRs!

**Comment by machour** [comment]
#724 got merged, closing this PR.

**Pull request closed without merging**